x-pack/plugins/security_solution/public/timelines/components/timeline/cell_rendering/default_cell_renderer.tsx

```diff
@@ -46,11 +46,14 @@ export const DefaultCellRenderer: React.FC<CellValueElementProps> = ({
   const styledContentClassName = isDetails
     ? 'eui-textBreakWord'
     : 'eui-displayInlineBlock eui-textTruncate';
+  const asPlainText = useMemo(() => {
```

Comment:
nit: consider hosting this `useMemo` to the top of the function before the first non-hook function invocation